libbpf-tools/drsnoop.c

```diff
@@ -31,18 +31,18 @@ const char *argp_program_bug_address = "<bpf@vger.kernel.org>";
 const char argp_program_doc[] =
 "Trace direct reclaim latency.\n"
 "\n"
-"USAGE: drsnoop [--help] [-p PID] [-t TID] [-d DURATION] [-e]\n"
+"USAGE: drsnoop [--help] [-p PID] [-t TID] [-d DURATION] [-E]\n"
 "\n"
 "EXAMPLES:\n"
 "    drsnoop         # trace all direct reclaim events\n"
 "    drsnoop -p 123  # trace pid 123\n"
 "    drsnoop -t 123  # trace tid 123 (use for threads only)\n"
 "    drsnoop -d 10   # trace for 10 seconds only\n"
-"    drsnoop -e      # trace all direct reclaim events with extended faileds\n";
+"    drsnoop -E      # trace all direct reclaim events with extended faileds\n";
 
 static const struct argp_option opts[] = {
 	{ "duration", 'd', "DURATION", 0, "Total duration of trace in seconds" },
-	{ "extended", 'e', NULL, 0, "Extended fields output" },
+	{ "extended", 'E', NULL, 0, "Extended fields output" },
```

Comment:
this seems to be libbpf-tools exclusive argument, but again, why changing it in the first place?